tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/Search.cs

```diff
@@ -1452,9 +1458,11 @@ private void DecodeMatterProperties(IDictionary<string, object> matterMetadata)
             matterMetadata[searchSettings.ManagedPropertyTitle] = DecodeValues(matterMetadata[searchSettings.ManagedPropertyTitle]);
             matterMetadata[searchSettings.ManagedPropertySiteName] = DecodeValues(matterMetadata[searchSettings.ManagedPropertySiteName]);
             matterMetadata[searchSettings.ManagedPropertyDescription] = DecodeValues(matterMetadata[searchSettings.ManagedPropertyDescription]);
-            matterMetadata[searchSettings.ManagedPropertyPracticeGroup] = DecodeValues(matterMetadata[searchSettings.ManagedPropertyPracticeGroup]);
-            matterMetadata[searchSettings.ManagedPropertyAreaOfLaw] = DecodeValues(matterMetadata[searchSettings.ManagedPropertyAreaOfLaw]);
-            matterMetadata[searchSettings.ManagedPropertySubAreaOfLaw] = DecodeValues(matterMetadata[searchSettings.ManagedPropertySubAreaOfLaw]);
+            var managedColumns = configuration.GetSection("ContentTypes").GetSection("ManagedColumns").GetChildren();
```

Comment:
Generic code to add taxonomy related columns to SPO managed properties to decode values